homeassistant/components/v2c/entity.py

```diff
@@ -0,0 +1,41 @@
+"""Support for V2C EVSE."""
+from __future__ import annotations
+
+from pytrydan import TrydanData
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import V2CUpdateCoordinator
+
+
+class V2CBaseEntity(CoordinatorEntity[V2CUpdateCoordinator]):
+    """Defines a base v2c entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: V2CUpdateCoordinator,
+        description: EntityDescription,
+    ) -> None:
+        """Init the V2C base entity."""
+        self.entity_description = description
+        super().__init__(coordinator)
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, coordinator.evse.host)},
```

Comment:
We don't allow an IP address as unique idenfier for devices or entities. What does host represent here?

Reply:
Changed to config_entry.entry_id